SIGs/GRC/publications/Confidential_Workload_Upgrade_Governance.md

```diff
@@ -38,7 +38,7 @@ a. Failure to handle intermediate states where only a subset of Workload instanc
 
 1. Remote Attestation Procedures (RATS) Architecture RFC: [https://datatracker.ietf.org/doc/rfc9334/](https://datatracker.ietf.org/doc/rfc9334/)  
 2. Key Words for Use in RFCs to Indicate Requirement Levels: [https://datatracker.ietf.org/doc/rfc2119/](https://datatracker.ietf.org/doc/rfc2119/)  
-3. Expectations of Ecosystem Participants [https://docs.google.com/document/d/1X10ymjFgUy5NGPKs6Z3F\_ERPwrgRuK23-9F2h3-LbWw/edit?usp=sharing](https://docs.google.com/document/d/1X10ymjFgUy5NGPKs6Z3F_ERPwrgRuK23-9F2h3-LbWw/edit?usp=sharing)  
+3. [Expectations of Ecosystem Participants](./Expectations_of_Ecosystem_Participants.md)  
```

Comment:
Ensure format is consistent across links: text followed by hyperlink, not just hyperlink